src/profile_manager/tests/test_views.py

```diff
@@ -176,6 +176,70 @@ def test_profile_detail__student_marks_button(self):
         response = self.client.get(reverse('profiles:profile_detail', args=[s_pk]))
         self.assertContains(response, 'View your Mark Calculations')
 
+    def test_profile_detail__xp_earned_by_tag_show_all_tags_on_profiles_true(self):
+        """ check if 'SiteConfig.get().show_all_tags_on_profiles = True' has ProfileDetail shows all the tags """
```

Comment:
This sentence makes no sense. perhaps:  ...shows all the tags on the student profile, instead of just tags where the student has earned XP

Reply:
done. also combined the two tests. heres the new doc string:
""" tests 'SiteConfig.get().show_all_tags_on_profiles' when true or false
        if true:
            checks if shows all the tags on the student profile, instead of just tags where the student has earned XP
        if false:
            checks if shows only tags where student has earned XP, instead of all tags
        """